solr/solr-ref-guide/ui-src/css/tabs.css

```diff
@@ -0,0 +1,103 @@
+.tabs {
```

Comment:
[Q] I came here to point out that this file should probably have an ASL 2.0 license header, but it turns out that none of the files in this directory have one.

Any other committers know whether this is intentional or accidental?  (Maybe the files weren't written by us, or we don't put license headers in UI files for size/network-transfer reasons? idk)

Reply:
I configured my IDE to put automatically on all files I create license headers and explicitly removed it here because I saw that the license header was missing in other files too. A clarification when to put a header would be great indeed.

Reply:
I helped commit it so I should probably have a better answer, but I have no idea why they don't have them. If this is a new file include it for now, and we will worry about the other ones later?

Reply:
Makes sense to me.  Looks like you already created SOLR-17365 to cover adding the missing licenses, thanks Houston!

Reply:
I would resolve this for now without adding to the file a license header yet, as your point @gerlowskija about size/network-transfer might be significant.

We may consider a minified license header for these files or configure Gulp to strip headers and add a license header to the minified files at build time, so that there are no multiple license headers in the files of the bundle. I am also not sure if the current gulp configuration does any cleanup or minification before bundling that may remove existing headers / comments.